tests/integration/testutil/common.go

```diff
@@ -155,36 +158,38 @@ func CreateTestInput(t *testing.T) TestInput {
 		rewardtypes.ModuleName:         nil,
 	}
 
+	govModAddress := authtypes.NewModuleAddress(govtypes.ModuleName).String()
+
 	paramsKeeper := paramskeeper.NewKeeper(appCodec, legacyAmino, keyParams, tKeyParams)
-	accountKeeper := authkeeper.NewAccountKeeper(appCodec, keyAcc, paramsKeeper.Subspace(authtypes.ModuleName), authtypes.ProtoBaseAccount, maccPerms)
-	bankKeeper := bankkeeper.NewBaseKeeper(appCodec, keyBank, accountKeeper, paramsKeeper.Subspace(banktypes.ModuleName), blackListAddrs)
+	accountKeeper := authkeeper.NewAccountKeeper(appCodec, keyAcc, authtypes.ProtoBaseAccount, maccPerms, sdk.GetConfig().GetBech32AccountAddrPrefix(), govModAddress)
+	bankKeeper := bankkeeper.NewBaseKeeper(appCodec, keyBank, accountKeeper, blackListAddrs, govModAddress)
 
 	var volunteerKeeper volunteerkeeper.Keeper
 	stakingKeeper := stakingkeeper.NewKeeper(
 		appCodec,
 		keyStaking,
 		accountKeeper,
 		bankKeeper,
-		paramsKeeper.Subspace(stakingtypes.ModuleName),
+		govModAddress,
 		&volunteerKeeper,
 	)
 
 	stakingParams := stakingtypes.DefaultParams()
 	stakingParams.BondDenom = sdk.DefaultBondDenom
 	stakingKeeper.SetParams(ctx, stakingParams)
 
-	mintKeeper := mintkeeper.NewKeeper(appCodec, keyMint, paramsKeeper.Subspace(minttypes.ModuleName), stakingKeeper, accountKeeper, bankKeeper, authtypes.FeeCollectorName)
+	mintKeeper := mintkeeper.NewKeeper(appCodec, keyMint, stakingKeeper, accountKeeper, bankKeeper, authtypes.FeeCollectorName, govModAddress)
 
 	distrKeeper := distrkeeper.NewKeeper(
 		appCodec,
-		keyDistr, paramsKeeper.Subspace(distrtypes.ModuleName),
+		keyDistr,
 		accountKeeper, bankKeeper, stakingKeeper,
-		authtypes.FeeCollectorName, blackListAddrs)
+		authtypes.FeeCollectorName, govModAddress)
 
-	slashingKeeper := slashingkeeper.NewKeeper(appCodec, keySlahsing, &stakingKeeper, paramsKeeper.Subspace(slashingtypes.ModuleName))
+	slashingKeeper := slashingkeeper.NewKeeper(appCodec, legacyAmino, keySlahsing, stakingKeeper, govModAddress)
 	slashingKeeper.SetParams(ctx, slashingtypes.DefaultParams())
 
-	volunteerKeeper = volunteerkeeper.NewKeeper(keyVolunteer, appCodec, &stakingKeeper, distrKeeper)
+	volunteerKeeper = volunteerkeeper.NewKeeper(keyVolunteer, appCodec, stakingKeeper, distrKeeper)
 
 	distrKeeper.SetFeePool(ctx, distrtypes.InitialFeePool())
 	distrParams := distrtypes.DefaultParams()
```

Comment:
DistributeParams was changed. It needs to remove `BaseProposerReward` & `BonusProposerReward`